Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -1223,13 +1262,18 @@ bool SkylanderPortal::CreateSkylander(const std::string& file_path, u16 m_sky_id
   // The BCC (Block Check Character)
   file_data[4] = file_data[0] ^ file_data[1] ^ file_data[2] ^ file_data[3];
 
-  // ATQA
+  // SAK
   file_data[5] = 0x81;
-  file_data[6] = 0x01;
 
-  // SAK
+  // ATQA
+  file_data[6] = 0x01;
   file_data[7] = 0x0F;
 
+  // Year of production
+  // the last 2 digits of the year the tag was produced, interpreted as hexadecimal
+  // 2011 example:
+  // file_data[0xF] = 0x11;
```

Comment:
Is that correct? 0x11 is 17 in decimal

Reply:
"the last 2 digits of the year the tag was produced, interpreted as hexadecimal". With interpreted as hexadecimal, I mean 2011 becomes 0x11, 2012 becomes 0x12, 2013 becomes 0x13, etc. So yeah. That's correct. Wording suggestions are always appreciated

Reply:
>Wording suggestions are always appreciated

Seems like [binary-coded decimal](https://en.wikipedia.org/wiki/Binary-coded_decimal)?